app/services/fees/charge_service.rb

```diff
@@ -59,6 +68,17 @@ def init_fees
     end
 
     def init_fee(properties:, group: nil)
+      if invoice.draft? && adjusted_fee(group) && !adjusted_fee(group).adjusted_display_name?
```

Comment:
Could you add a comment, or extract the condition in a method, to help understand what we are doing here?